### PR TITLE
Set Default Username and Password

### LINKED ISSUE #1: Set Default Username and Password
The user name and password can be set via the SQL dump provided.
Defaults to use:  username: admin, password: P@ssw0rd
Steps:
1. Use MD5 to hash the word "P@ssw0rd".
2. Write an insert query after the SQL command to create table Users in the ce.sql_Schema. The insert query should have 'admin' as user role and MD5 hash obtained earlier as password.
3. Commit the file.

Also, Mention the default username and password in the README file.

### PR DESCRIPTION
**Pull request by luckymurari**
Fixes #1

### CODE REVIEW

**Review by vishnudas-raveendran — APPROVED** [review]
Ok.

File: WCMST1.sql_schema [hunk 1]
```diff
@@ -154,7 +154,7 @@ CREATE TABLE IF NOT EXISTS `users` (
 ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci AUTO_INCREMENT=10 ;
 
 
-
+INSERT INTO `users` (`id`, `username`, `password`, `last_logged`) VALUES (NULL, 'admin', MD5('P@ssw0rd'), current_timestamp());
```
Comment (sebinbenjamin) [inline WCMST1.sql_schema:157]:
```
CREATE TABLE IF NOT EXISTS `users` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `username` varchar(30) COLLATE utf8_unicode_ci NOT NULL,
  `password` varchar(50) COLLATE utf8_unicode_ci NOT NULL,
  `last_logged` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  PRIMARY KEY (`id`),
  UNIQUE KEY `username` (`username`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci AUTO_INCREMENT=10 ;

```
@vishnudas-raveendran @luckymurari If I understand correctly, `id` column here has a `NOT NULL` constrain and we would get an error.

Comment (sebinbenjamin) [reply WCMST1.sql_schema:157]: I think we could skip the `id` and `last_logged ` columns as they will be auto-generated/default.

**Review by sebinbenjamin — COMMENTED** [review]
`INSERT INTO 'users' ('username', 'password') VALUES ('admin', MD5('P@ssw0rd'));`

File: WCMST1.sql_schema (hunk 1 above)
Comment (vishnudas-raveendran) [reply WCMST1.sql_schema:157]: Yes true. we must skip that. Oversight on my part, sorry. Reverted pull request #13

**Comment by vishnudas-raveendran** [comment]
@sebinbenjamin Thanks for that review. Oversight on my part.
Have reverted that commit with pull request #13 .

**Pull request merged**